frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ClusterDetails/ClusterDetails.test.tsx

```diff
@@ -1314,7 +1314,7 @@ describe('ClusterDetails', () => {
   test('overview page opens logs', async () => {
     const nocks: Scope[] = [nockListHiveProvisionJobs()]
     window.open = jest.fn()
-    await clickByText('View logs', 1)
+    await clickByText('View logs', 0)
```

Comment:
Change in behavior in the log button requires change in this test, namely, when there are posthook jobs defined but no prehook jobs defined, we no longer render the prehook link. We are still rendering the documentation link when no hooks are defined as well.